mpt/impl/imap-mailbox/cyrus/pom.xml

```diff
@@ -62,7 +62,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>20.0</version><!--$NO-MVN-MAN-VER$-->
+            <version>25.1-jre</version><!--$NO-MVN-MAN-VER$-->
```

Comment:
this line is useless now that all components are using the same guava version, no?